### PR TITLE
[Tizen/API] Update Tizen API to comply with ACR process

### PR DESCRIPTION
**Pull request by again4you**
This PR modifies the API and its related thing to comply with Tizen ACR process.
Detailed items are as below.

* Make a consistency in naming of GStreamer
* Make a verb for the third person singular
* Add 'remarks' information for specific functions

### Related Issue
* https://github.com/nnsuite/nnstreamer/issues/1435

### Self evaluation:
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

Signed-off-by: Sangjung Woo <sangjung.woo@samsung.com>

### CODE REVIEW

File: tizen-api/include/nnstreamer.h [hunk 1]
```diff
@@ -215,8 +216,8 @@ typedef void (*nns_sink_cb)
 int nns_pipeline_construct (const char *pipeline_description, nns_pipeline_h *pipe);
 
 /**
- * @brief Destroy the pipeline
- * @detail Use this function to destroy the pipeline constructed with nns_construct_pipeline.
+ * @brief Destroys the pipeline
+ * @detail Uses this function to destroy the pipeline constructed with nns_construct_pipeline.
```
Comment (wooksong) [inline tizen-api/include/nnstreamer.h]: I do not know what is the convention for Tizen APIs used here, but other comments mentioned function names seems to be followed by (). Is it ok without () here?

File: tizen-api/include/nnstreamer.h [hunk 2]
```diff
@@ -227,8 +228,8 @@ int nns_pipeline_construct (const char *pipeline_description, nns_pipeline_h *pi
 int nns_pipeline_destroy (nns_pipeline_h pipe);
 
 /**
- * @brief Get the state of pipeline
- * @detail Get the state of The pipeline handle returned by nns_construct_pipeline (pipe).
+ * @brief Gets the state of pipeline
+ * @detail Gets the state of The pipeline handle returned by nns_construct_pipeline (pipe).
```
Comment (wooksong) [inline tizen-api/include/nnstreamer.h]: The -> the

Comment (wooksong) [reply tizen-api/include/nnstreamer.h]: nns_constuct_pipeline() or nns_constuct_pipeline (). Which one is right? Are both of them fine?

File: tizen-api/include/nnstreamer.h (hunk 1 above)
Comment (again4you) [reply tizen-api/include/nnstreamer.h]: According to the guide, a verb after `@brief` should be a third person singular format without `()`. You can see a code of autofill module, which is recently merged into Tizen.
* https://git.tizen.org/cgit/platform/core/uifw/autofill/tree/include/autofill.h?h=tizen#n208

File: tizen-api/include/nnstreamer.h (hunk 2 above)
Comment (again4you) [reply tizen-api/include/nnstreamer.h]: Nice catch! I will.
* The -> the

> nns_constuct_pipeline() or nns_constuct_pipeline (). Which one is right? Are both of them fine?

What about adding space between a name and `()`?
We already have been used this way in our codes.

**Comment by again4you** [comment]
### Submit v2
* fix the typo (The -> the)


@wooksong, Could you go over this work and merge it?

**Pull request merged**